### ISSUE TITLE
Docker Desktop start-up prompt suitable for the visually impared

### ISSUE THREAD

**Issue by GadgetSteve** [docker/roadmap#528]
**Tell us about your request**
On at least Windows the Docker Desktop start-up prompt, (which comes up after windows start-up), _while suitable for screen readers_, is not operable by keyboard as the buttons do not take focus and lack keyboard shortcuts. This can be a problem for visually impaired people as mouse operations can be an issue.

Please consider at least one of:

 - Using a dialogue which allows focus on the buttons with the default being Yes and tab moving to No.
 - Binding and Displaying some keyboard shortcuts in the dialogue
 - Allowing docker desktop to be set to always run without prompting on start-up.

**Which service(s) is this request for?**
Docker Desktop Start-up

**Tell us about the problem you're trying to solve. What are you trying to do, and why is it hard?**
Personally I'm always frustrated when I am forced to use the mouse but for those with visual impairment and some types of mobility problems this is much more of an issue.

**Are you currently working around the issue?**
Forced to use the mouse.

**Additional context**

**Comment by mark-dr** [comment]
Hi @GadgetSteve, and thanks for the detailed and considered feedback! Just to help us understand, could you clarify (ideally with screenshots, if it's possible) what you're referring to as the start-up prompt?

Separately, re this point:

> Allowing docker desktop to be set to always run without prompting on start-up

The below two items are available in the settings interface of Docker Desktop:

- Start Docker Desktop when you log in
- Open Docker Dashboard at startup

Does the combination of those help solve your problem?
